2_intermediate/chapter11/solutions/remove_duplicates.py

```diff
@@ -0,0 +1,31 @@
+# Write a function called remove_duplicates
+# The sole parameter of the function should be a list
+# The function should look through a list,
+# Find all duplicate elements, and remove them
+# Sort the resulting list
+# Hint: To sort a list, use sorted(list)
+# Another hint: Use list.count(element)
+# To count the number of times that element appears
+
+# Example: array = [1,1,2,5,4,6,12,3,4,6]
+# Result should print [1,2,3,4,5,6,12]
+
+# Write code here
+
+list1 = [1, 1, 2, 5, 4, 6, 12, 3, 4, 6]  # Define your list
+
+
+def remove_duplicate(array):
+    for i in array:
```

Comment:
It's possible, even probable, that Python gets all the elements in ```array``` only once, which is why when you call ```array.remove()```, ```array``` changes but what the loop is iterating over does not, which causes your solution to crash.

Reply:
Hmm...I'll make a new solution and check if it works, and then I'll fix it. Thanks for the input.